db/migrate/20151115221849_add_subscribed_to_participant_activity_to_users.rb

```diff
@@ -0,0 +1,10 @@
+class AddSubscribedToParticipantActivityToUsers < ActiveRecord::Migration
+  def up
+    add_column :users, :subscribed_to_participant_activity, :boolean, default: false
+    User.update_all(subscribed_to_participant_activity: false)
```

Comment:
https://github.com/cobudget/cobudget-api/commit/f09eef71613959fb2ad6f7d3ea6a231989d93e8f

damn took me super long to figure out that you can't `validates_presence_of` on `boolean` properties